sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -175,7 +175,7 @@ class CustomerIO internal constructor(
         private val modules: MutableMap<String, CustomerIOModule<out CustomerIOModuleConfig>> =
             mutableMapOf()
         private var logLevel: CioLogLevel = SDKConstants.LOG_LEVEL_DEFAULT
-        internal var overrideDiGraph: CustomerIOComponent? = null // set for automated tests
+        public var overrideDiGraph: CustomerIOComponent? = null // set for automated tests
```

Comment:
Even though it is in builder and I don't expect customers playing around with this a lot, but I would still recommend to annotate it with `InternalCustomerIOApi` and add slightly more descriptive comment.